controller/src/test/java/io/pravega/controller/task/KeyValueTable/TableMetadataTasksTest.java

```diff
@@ -137,6 +164,11 @@ public void testCreateKeyValueTable() throws ExecutionException, InterruptedExce
                 e -> Exceptions.unwrap(e) instanceof RuntimeException);
     }
 
+    private long getTimerMillis(String timerName) {
```

Comment:
Same here, this is being repeated everywhere.

Reply:
moved to a common place in a new class